features/gba-dev/ouders.feature

```diff
@@ -0,0 +1,234 @@
+# language: nl
+
+@post-assert
+Functionaliteit: Ouders van een persoon raadplegen
+  Van een persoon kunnen ouders worden geleverd.
+  Dit bevat gegevens over de familierechtelijke betrekking plus enkele identificerende eigenschappen van de ouder.
+
+@gba
```

Comment:
de @gba is tag niet meer nodig

Reply:
Ik heb tot nu toe alleen de @gba tags verwijderd als ze voor een Scenario stonden. Moeten ze ook verwijderd worden als ze voor een Rule staan?